CHANGES.rst

```diff
@@ -1,3 +1,13 @@
+FlexibleSUSY 2.X.X [XXX]
+==================================
+
+New features
+------------
+
+* Added calculation of Barr-Zee type contributions to g-2.
```

Comment:
please add "of the muon"

Reply:
Done in 739c04976b72a0e434b881ff2234f0dd780fdc47.